snuba/datasets/entities/events.py

```diff
@@ -73,16 +115,85 @@ def __init__(self, custom_mappers: Optional[TranslationMappers] = None) -> None:
         schema = storage.get_table_writer().get_schema()
         columns = schema.get_columns()
 
+        events_pipeline_builder = SimplePipelineBuilder(
+            query_plan_builder=SelectedStorageQueryPlanBuilder(
+                selector=EventsQueryStorageSelector(
+                    mappers=event_translator
+                    if custom_mappers is None
+                    else event_translator.concat(custom_mappers)
+                )
+            ),
+        )
+
+        errors_pipeline_builder = SimplePipelineBuilder(
+            query_plan_builder=SelectedStorageQueryPlanBuilder(
+                selector=ErrorsQueryStorageSelector(
+                    mappers=errors_translators
+                    if custom_mappers is None
+                    else errors_translators.concat(custom_mappers)
+                )
+            ),
+        )
+
+        def selector_func(_query: Query) -> Tuple[str, List[str]]:
```

Comment:
Could we reuse the callback we already have in discover. It seems you are making changes often, it would be easy to miss changes on one side.

Reply:
Yeah, I guess there's no reason the two callbacks can't be the same